test/fuzz/utility.h

```diff
@@ -87,6 +89,8 @@ inline test::fuzz::Headers toHeaders(const Http::HeaderMap& headers) {
 
 inline TestStreamInfo fromStreamInfo(const test::fuzz::StreamInfo& stream_info,
                                      const Ssl::MockConnectionInfo* connection_info) {
+  // Set mocks' default string return value to be an empty string.
```

Comment:
I don't feel strongly but would it be better to just specify the ON_CALL directly in the broken mock to be a bit more specific?

Reply:
I'd have to ON_CALL each of the string related field extractors in MockConnectionInfo (https://github.com/envoyproxy/envoy/blob/a49c06b1789b398ac4c9207cc5bd0cd369342e85/test/mocks/ssl/mocks.h#L33). I figured a default value would cover all of them in case we want to add field in StreamInfo and specifically ON_CALL the functions there.